src/learningReflections/css/learningReflections.css

```diff
@@ -173,6 +173,11 @@ body {
     color: var(--site-blue);
 }
 
+.sjrk-st-context-link {
```

Comment:
There's another css definition for `.sjrk-st-context-link` at [line 92](https://github.com/BlueSlug/sjrk-story-telling/blob/SJRK-201/src/learningReflections/css/learningReflections.css#L92). Is it possible to combine these two?